File: Strings/__2000__/2651/2651.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main(){

    string input;
    getline(cin, input);

    transform(input.begin(), input.end(), input.begin(), ::tolower);

    cout << (input.find("zelda") != string::npos ? "Link Bolado" : "Link Tranquilo") << endl;

    return 0;
}

File: Strings/__2000__/2651/2651.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);

        String input = in.nextLine().toLowerCase();

        System.out.println(input.contains("zelda") ? "Link Bolado" : "Link Tranquilo");
        in.close();
    }
}